test/sharness/t0066-migration.sh

```diff
@@ -30,15 +41,23 @@ test_expect_success "output looks good" '
   grep "Please get fs-repo-migrations from https://dist.ipfs.io" false_out
 '
 
+# The migrations will succeed, but the daemon will still exit with 1 because
+# the fake migrations do not update the repo version number.
+#
+# If run with real migrations, the daemon continues running and must be killed.
 test_expect_success "ipfs daemon --migrate=true runs migration" '
-  test_expect_code 1 ipfs daemon --migrate=true > true_out
+  test_expect_code 1 ipfs daemon --migrate=true > true_out 2>&1
```

Comment:
What is this change doing for us?

Reply:
That was needed because PR #8054 was not merged yet.  Some of the output was now going to stderr that was expected to go to stdout.  That made a subsequent test, that scanned `true_out` fail, but for the wrong reasons.

Reply:
If I rebase on top of #8054, then that change can go away.

Reply:
Sounds good, let's rebase now that the bug is fixed and keep it as is.

Reply:
Done.  All tests passing.